lib/db/helpers/index.js

```diff
@@ -27,7 +31,206 @@ let findOne = (profileID) => {
   })
 }
 
+let findRoomByName = (allrooms, title) => {
+  //findIndex is the new es6 method
+  let findRoom = allrooms.findIndex((element, index, array) => {
+    if (element.title === title) {
```

Comment:
You could just return element.title === title instead of the if/else statement :smiley: